source/reference/minio-mc-admin/mc-admin-cluster-iam-export.rst

```diff
@@ -0,0 +1,71 @@
+.. _minio-mc-admin-cluster-iam-export:
+
+===============================
+``mc admin cluster iam export``
+===============================
+
+.. default-domain:: minio
+
+.. mc:: mc admin cluster iam export
+
+Description
+-----------
+
+.. versionadded:: RELEASE.2022-06-26T18-51-48Z
+
+.. start-mc-admin-cluster-iam-export-desc
+
+The :mc:`mc admin cluster iam export` command exports :ref:`IAM <minio-authentication-and-identity-management>` metadata for use with the :mc:`mc admin cluster iam import` command.
+
+.. end-mc-admin-cluster-iam-export-desc
+
+The command always saves the output as ``ALIAS-iam-metadata.zip``, where ``ALIAS`` is the :mc:`alias <mc admin cluster iam export ALIAS>` of the MinIO deployment.
+
+. tab-set::
+
+   .. tab-item:: EXAMPLE
+
+      The following command exports all iam metadata for the ``myminio`` deployment.
+  
+      .. code-block:: shell  
+         :class: copyable 
+  
+         mc admin cluster iam export myminio
+
+   .. tab-item:: SYNTAX
+
+      The command has the following syntax: 
+  
+      .. code-block:: shell  
+         :class: copyable 
+  
+         mc [GLOBALFLAGS] admin cluster iam export  \
+                                            ALIAS[/BUCKET] \
+                                             
+      .. include:: /includes/common-minio-mc.rst
+         :start-after: start-minio-syntax
+         :end-before: end-minio-syntax
+
+Starting with :mc-release:`RELEASE.2023-05-04T18-10-16Z`, :mc:`mc admin cluster iam export` adds support for aliases ending with a trailing slash ``ALIAS/``.
+Prior to this release, the command would fail when provided a trailing slash.
+
+Parameters
+~~~~~~~~~~
+
+.. mc-cmd:: ALIAS
+   :required:
+
+   The :ref:`alias <alias>` of the MinIO deployment.
+
+.. mc-cmd:: BUCKET
```

Comment:
How can you export IAM for a bucket?
The CLI has `target`, so it's unclear if it is just an alias, or an alias/bucket.
But alias makes more sense.

Reply:
This is a copy-pasta mistake.